hexa/databases/graphql/schema.graphql

```diff
@@ -0,0 +1,30 @@
+type Database {
+  tables(page:Int, perPage:Int): DatabaseTablePage!
+}
+
+type TableColumn {
+  name: String!
+  type: String!
+  sample: [[String]]
+}
+
+
+type DatabaseTable{
+  name: String!
+  totalCount: Int!
+  columns: [TableColumn!]!
+  sample:[[String]]
+  database: Database!
+}
+
+type DatabaseTablePage {
+  pageNumber: Int!
+  totalPages: Int!
+  totalItems: Int!
+  items: [DatabaseTable!]!
+}
+
+
+extend type Query{
+  databaseTable(id:String!,databaseId:String!): DatabaseTable @loginRequired
```

Comment:
This should be included in the `Workspace` type and without the `databaseId` argument (since you would receive a workspace)